product_state/__manifest__.py

```diff
@@ -1,19 +1,21 @@
 # Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
-    'name': "Product State",
-    'summary': """
+    "name": "Product State",
+    "summary": """
         Module introducing a state field on product template""",
-    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
-    'website': "https://github.com/OCA/product-attribute",
-    'category': 'Product',
-    'version': '12.0.1.0.0',
-    'license': 'AGPL-3',
-    'depends': [
-        'product',
+    "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "category": "Product",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["sale", "product"],
+    "data": [
+        "data/product_state_data.xml",
+        "security/ir.model.access.csv",
+        "views/product_views.xml",
     ],
-    'data': [
-        'views/product_views.xml',
-    ],
-    'application': True,
+    "application": True,
```

Comment:
IMO this is not an App, it is a technical feature
```suggestion
```

Reply:
I agree, but It is set as an application https://github.com/OCA/product-attribute/tree/12.0/product_state
So, maybe it is better to change this in the v13 migration #634